cmd/av/pr_queue.go

```diff
@@ -0,0 +1,110 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/aviator-co/av/internal/actions"
+	"github.com/aviator-co/av/internal/avgql"
+	"github.com/shurcooL/graphql"
+	"github.com/spf13/cobra"
+)
+
+var prQueueFlags struct {
+	SkipLine 	bool
+	Targets		string
+}
+
+var prQueueCmd = &cobra.Command{
+	Use:          "queue",
+	Short:        "queue a pr for the current branch",
+	SilenceUsage: true,
+	Args:         cobra.NoArgs,
+	// error or reterr from emperror.dev/errors here?
+	RunE: func(cmd *cobra.Command, args []string) error {
+		variables, err := getMutationVariables()
+		if err != nil {
+			return err
+		}
+
+
+		// I have a feeling this would be better written inside of av/internals
+		client := avgql.NewClient()
+
+		var mutation struct {
+			QueuePullRequest struct {
+				QueuePullRequestPayload struct {
+					PullRequest	graphql.String
```

Comment:
I think we need to actually finish the backend before merging this, because converting the `pullRequest` field from string to an actual GraphQL object would be a breaking change.

Reply:
Got it

Reply:
Im grinding on this, bitten by circular dependencies. I think in the sake of shipping the best move is to simply breakup queue.py on the backend.

Reply:
Done; see: https://github.com/aviator-co/mergeit/pull/3053